CotEditor/Storyboards/mul.lproj/Main.xcstrings

```diff
@@ -9293,6 +9869,12 @@
             "value" : "Converter texto alfanumérico para largura total"
           }
         },
+        "ru" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Преобразовать буквенно-цифровой текст в полную ширину"
```

Comment:
Could you also update this label along with “Convert to Full Width” and “Convert to Half Width”?